cypress-custom/integration/swapMod.ts

```diff
@@ -32,17 +32,23 @@ describe('Swap (mod)', () => {
   })
 
   it('can enter an amount into output', () => {
-    cy.get('#swap-currency-output .token-amount-input')
+    // first, clear the INPUT currency input field
+    // as it is auto filled with "1"
+    cy.get('#swap-currency-input .token-amount-input')
+      .clear()
+      .get('#swap-currency-output .token-amount-input')
```

Comment:
I would expect the comment to continue here:
`// then, clear the out put...`

Also, which input will be filled here?
input or output?

Reply:
By looking at the test name, output. 

Both inputs have the same effect when you delete the content, but here we are trying to emulate the user behaviour, so its more rational that you also delete the output

Reply:
output? it's in the test name or what do you guys mean
```
it('can enter an amount into output', () => {
```
i'll add the comment that it's clearing the output as well, which is doing in line 40. First we clear the default input amount (`1`) - hence the comment, then we do as normal, clear output and type etc

Reply:
I just got confused with the chained `.get`, got it now